workers/unity/Packages/com.improbable.gdk.tools/.CodeGenerator/GdkCodeGenerator/src/Generation/Model/Details/DetailsStore.cs

```diff
@@ -0,0 +1,144 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using Improbable.Gdk.CodeGeneration.Model.SchemaBundleV1;
+
+namespace Improbable.Gdk.CodeGenerator
+{
+    public class DetailsStore
+    {
+        public IReadOnlyDictionary<Identifier, UnityTypeDetails> Types { get; }
+        public IReadOnlyDictionary<Identifier, UnityEnumDetails> Enums { get; }
+        public IReadOnlyDictionary<Identifier, UnityComponentDetails> Components { get; }
+        public readonly ReadOnlyDictionary<Identifier, bool> BlittableMap;
```

Comment:
You could also use a `HashSet` and `Contains`

Reply:
Yes you are correct.